### ISSUE TITLE
All prices are zero?

### ISSUE THREAD

**Issue by mareru** [ovcharski/playwright-e2e#2]
It seems a bit odd that your checking the e-commerce site, but the pricing doesn't seem relevant in your use case. Why is that so? :)

**Comment by ovcharski** [comment]
You're right, I hadn't thought about that. I'll think about adding some tests in the next days.
